client/webserver/site/src/js/locales.ts

```diff
@@ -149,7 +150,8 @@ export const enUS: Locale = {
   [ID_INVALID_DATE_ERR_MSG]: 'error: invalid date or time',
   [ID_NO_ARCHIVED_RECORDS]: 'No archived records found',
   [ID_DELETE_ARCHIVED_RECORDS_RESULT]: 'Message: {{ nRecords }} archived records has been deleted',
-  [ID_ARCHIVED_RECORDS_PATH]: 'File Location: {{ path }}'
+  [ID_ARCHIVED_RECORDS_PATH]: 'File Location: {{ path }}',
+  [ID_ORDER_PENDING]: 'pending'
```

Comment:
Having tested this, I think it would be better to show this as "submitting" rather than "pending".  In a different sense, "epoch" orders are also "pending", so let's be more specific.

Reply:
okay.